res/win-icon.rc

```diff
@@ -0,0 +1 @@
+IDI_ICON1 ICON DISCARDABLE "windows-icon.ico"
```

Comment:
I think this should be "windows_icon.ico". My VS project would fail at compiling otherwise